fix_scope.py

```
@@ -0,0 +1,58 @@
+import sys
+import os
+
+# Fix the config.py to allow any target in CTF mode
+config_path = 'cybershell/config.py'
+
+with open(config_path, 'r') as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)